aviary/mission/flops_based/ode/takeoff_ode.py

```diff
@@ -34,33 +29,47 @@ def initialize(self):
         options = self.options
 
         options.declare(
-            'num_nodes', default=1, types=int,
-            desc='Number of nodes to be evaluated in the RHS')
+            'num_nodes',
+            default=1,
+            types=int,
+            desc='Number of nodes to be evaluated in the RHS',
+        )
 
         options.declare(
-            'aviary_options', types=AviaryValues,
-            desc='collection of Aircraft/Mission specific options')
+            'aviary_options',
+            types=AviaryValues,
+            desc='collection of Aircraft/Mission specific options',
+        )
 
         self.options.declare(
-            'subsystem_options', types=dict, default={},
-            desc='dictionary of parameters to be passed to the subsystem builders')
+            'subsystem_options',
+            types=dict,
+            default={},
+            desc='dictionary of parameters to be passed to the subsystem builders',
+        )
 
         self.options.declare(
             'core_subsystems',
-            desc='list of core subsystem builder instances to be added to the ODE'
+            desc='list of core subsystem builder instances to be added to the ODE',
         )
         self.options.declare(
-            'external_subsystems', default=[],
-            desc='list of external subsystem builder instances to be added to the ODE')
+            'external_subsystems',
+            default=[],
+            desc='list of external subsystem builder instances to be added to the ODE',
+        )
 
         options.declare(
             'friction_key',
             desc='current friction coefficient key, '
-            'either rolling friction or braking friction')
+            'either rolling friction or braking friction',
+        )
 
         options.declare(
-            'climbing', default=False, types=bool,
-            desc='mode of operation (ground roll or flight)')
+            'climbing',
+            default=False,
+            types=bool,
+            desc='mode of operation (ground roll or flight)',
```

Comment:
This option seems to get passed around several times, so updating either will require hunting around to find all the places it was used. IMO the GASP mission components need a cleaning pass anyway, so that could be part of that separate effort